src/execution/__tests__/lists-test.ts

```diff
@@ -37,6 +37,7 @@ describe('Execute: Accepts any iterable as list value', () => {
   });
 
   it('Accepts function arguments as a List value', () => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
we had setting for it here: [`1ec501c` (#2828)](https://github.com/graphql/graphql-js/pull/2828/commits/1ec501c005fda0ca03614db8ad688a855ebbce0d#diff-9e1ecc14c733bb1ae2e523089f1262ac6ffccbcf950487ee0984403603550e57L630)
You need to remove 1ec501c from PR and just fix eslint errors here.